src/models/aggregation_models.rs

```diff
@@ -4,12 +4,13 @@
 use super::engine_models::EngineError;
 use serde::{Deserialize, Serialize};
 use smallvec::SmallVec;
-
+#[cfg(feature = "use-synonyms-search")]
+use thesaurus::synonyms;
 /// A named struct to store the raw scraped search results scraped search results from the
 /// upstream search engines before aggregating it.It derives the Clone trait which is needed
 /// to write idiomatic rust using `Iterators`.
 /// (href url in html in simple words).
-#[derive(Clone, Serialize, Deserialize)]
+#[derive(Clone, Serialize, Deserialize, Debug)]
```

Comment:
```suggestion
#[derive(Clone, Serialize, Deserialize)]
```

I think you forgot to remove the debug trait from here. So I would suggest removing it. :slightly_smiling_face: .